### DISCUSSION TITLE
Changing the Max SoC when doing a full charge

### DISCUSSION THREAD

**Discussion by booti101** [nathanmarlor/foxess_em#369] [General]
Is there any way to change the Max SoC value when you have set the switch.full_charge entity please?
I regularly charge the battery overnight to full so that i can export all my solar and have battery capacity for the evening, but as the max SoC is always 100%, it will be putting all the strain on the battery to fill it to the max.

Would it be possible to adjust the max SoC value when setting a 'Full Charge'?

**Comment by idpandjab** [comment]
If you have FoxEss - Modbus you can do it in there. Not sure if this overwrites it though, never tried it.![image](https://github.com/user-attachments/assets/9a538ff6-f90d-4546-a971-bc81eb83aedd)

**Comment by booti101** [comment]
OK, I'll have a play and see if i can create an automation that sets the Min SoC (on grid) value to 90% at 00:01 and puts it back to 10% at 05:30.
